### PR TITLE
user profile page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
check this preview for better understanding
Animation2

### CODE REVIEW

File: product-listing/products.css [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+.search-bar{
+    outline: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    padding: 0.6rem 0.8rem;
+    margin: 0.5rem;
+}
 .middleSection{
     margin-top: 80px;
```
Comment (t007rushi) [inline product-listing/products.css]: maintain consistent units just suggestion

File: user profile/user_profile.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+}
+.wrapper nav label{
+    display: block;
+    height: 100%;
+    width:100%;
+    text-align: center;
+    cursor: pointer;
+    line-height: 50px;
+    color:var(--primary-light);
+    font-family: 17px;
```
Comment (Namrata-J) [inline user profile/user_profile.css]: what's this font-family 17px used for!!?

**Review by Namrata-J — APPROVED** [review]

**Review by t007rushi — APPROVED** [review]
Increase Heading Size of titles in user profile tabs

File: product-listing/products.css (hunk 1 above)
Comment (vaibhav18matere) [reply product-listing/products.css]: I try to use rem for smaller values and px for bigger one? is it good approach? also can i use rem for bigger values as well?

File: user profile/user_profile.css (hunk 2 above)
Comment (vaibhav18matere) [reply user profile/user_profile.css]: It's not needed,  will remove that. Thank you for bringing it to my notice.

**Review by abhijeetekad — APPROVED** [review]
This Iooks great. I will make this section in my ecom as well

**Comment by vaibhav18matere** [comment]
> This Iooks great. I will make this section in my ecom as well

Thank you, Abhi

**Pull request merged**